src/Symfony/Component/Notifier/Bridge/OneSignal/OneSignalTransport.php

```diff
@@ -77,7 +77,16 @@ protected function doSend(MessageInterface $message): SentMessage
 
         $options = $options?->toArray() ?? [];
         $options['app_id'] = $this->appId;
-        $options['include_player_ids'] = [$recipientId];
+        if ($options['is_external_user_id'] ?? false) {
+            $options['include_aliases'] = [
+                'external_id' => [$recipientId],
+            ];
+            $options['target_channel'] = 'push';
+            unset($options['is_external_user_id']);
+        } else {
+            // include_player_ids has been deprecated since OneSignal API Version 11
+            $options['include_subscription_ids'] = [$recipientId];
```

Comment:
include_player_ids or include_subscription_ids ? 🤔

Reply:
include_player_ids is the old one, I just thought I'd mention it there in case anyone is wondering about the change

Reply:
No need for the moment, it's just more confusing